lib/explorer/query.ex

```diff
@@ -409,18 +419,14 @@ defmodule Explorer.Query do
   end
 
   defp traverse({left, meta, right}, vars, state) do
-    cond do
-      Kernel.and(
-        Kernel.and(is_atom(left), is_list(right)),
-        special_form_defines_var?(left, right)
-      ) ->
-        raise ArgumentError,
-              "#{left}/#{length(right)} is not currently supported in Explorer.Query"
-
-      true ->
-        {left, vars} = traverse(left, vars, state)
-        {right, vars} = traverse(right, vars, state)
-        {{left, meta, right}, vars}
+    K.if is_atom(left)
+         |> K.and(is_list(right))
+         |> K.and(special_form_defines_var?(left, right)) do
+      raise ArgumentError, "#{left}/#{length(right)} is not currently supported in Explorer.Query"
+    else
+      {left, vars} = traverse(left, vars, state)
+      {right, vars} = traverse(right, vars, state)
+      {{left, meta, right}, vars}
```

Comment:
Are these changes related to this PR or is it just changing the style? If so, can we do it in a separate PR? :)

Reply:
Haha yes absolutely, sorry for the clutter!

Reply:
[6dd174b](https://github.com/elixir-explorer/explorer/pull/989/commits/6dd174bb4ec53bf4180c9b71aa97e7e480befb5b)